### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by soymariomoreno**
Hey, I have added translation in Spanish

### CODE REVIEW

File: resources/lang/esp.ini [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+tool.wand.lore.3 = "Use //togglewand  para alternar su funcionalidad"
+tool.wand.disabled = "La herramienta de varita está deshabilitada. Use //togglewand para volver a habilitarla"
+tool.wand.setenabled = "¡La herramienta de varita ahora es {% 0}!"
+; debug tool
+tool.debug = "Herramienta de depuración"
+tool.debug.lore.1 = "Haga clic izquierdo en un bloque para obtener información"
+tool.debug.lore.2 = "Como el nombre y los valores de daño de un bloque"
+tool.debug.lore.3 = "Use //toggledebug para alternar su funcionalidad"
+tool.debug.disabled = "La herramienta de depuración está deshabilitada. Use //toggledebug para volver a habilitarla"
+tool.debug.setenabled = "¡La herramienta de depuración ahora es {%0}!"
```
Comment (inxomnyaa) [inline resources/lang/esp.ini:86]: Just a question, what is this upside down "!" - have seen this quite often already :P

**Review by inxomnyaa — APPROVED** [review]
💯 Fits perfectly

File: resources/lang/esp.ini (hunk 1 above)
Comment (soymariomoreno) [reply resources/lang/esp.ini:86]: They are used to indicate surprise, amazement, joy, supplication :)

Comment (inxomnyaa) [reply resources/lang/esp.ini:86]: Pretty cool, thanks for letting me know :D

**Pull request merged**